### DISCUSSION TITLE
Status of client-cpp

### DISCUSSION THREAD

**Discussion by jcar87** [opentdf/platform#1819] [Platform Q&A] [answered]
Hi there. I'm a maintainer at Conan Center, and we have had reports that [https://github.com/opentdf/client-cpp] is no longer visible - so any potential user would not be able to build this from source anymore.

What's the status of the `client-cpp` project?
The documentation for the go package also makes reference to what is now a broken link: https://pkg.go.dev/github.com/opentdf/client-go#section-readme

* What should users do that were relying on the C++ implementation?
* In case the `client-cpp` repository is retired/unmaintained - is there any chance it could be made visible again, perhaps as an "archived" repository with a readme that makes this clear? That way the legacy sources can still be downloaded, for anyone who ever did rely on this?

**Comment by damorris25** [comment, marked as answer]
I don't believe the archived version will be useful as the client-cpp is incompatible with the current platform. If this is still an issue for someone we can consider your suggest to mark it as archived.
